week-3-practise/level-2/1-map.js

```diff
@@ -12,7 +12,12 @@ var londonLocations = [
   ["Greenwich", "tube", "bus", "river boat"]
 ];
 
-var locationsByBoat;
+var locationsByBoat = londonLocations.map(prova);
```

Comment:
This does not produce the expected output. The output is currently `[undefined, "London Bridge", undefined, "Greenwich"]`. You need to use filter to remove undefined entries